### DISCUSSION TITLE
One of my friend account is removed or hacked can you please help to reinitiate it.

### DISCUSSION THREAD

**Discussion by papayacoders-dev** [community/community#86688] [Other Feature Feedback, Questions, & Ideas] [answered]
Guyz Help me, it something that should not happen but happened. My github account is hacked or removed don't have idea. there is lot of things i have on my github also some library that is used by others. All gone.
![WhatsApp Image 2024-01-12 at 1 15 41 AM](https://github.com/community/community/assets/153585357/2be35e36-ab77-49d7-b01f-0f54bb6403a3)
![WhatsApp Image 2024-01-12 at 1 15 14 AM](https://github.com/community/community/assets/153585357/5bd55a31-f19d-46f0-ac44-4d97f6e21d1f)
![Screenshot 2024-01-12 005611](https://github.com/community/community/assets/153585357/4eedca1c-c9a0-4c5b-bf81-4d32694282ce)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @papayacoders-dev, 

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact) as the Community is unable to assist nor escalate in these situations. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
